doc/develop/microscope.rst

```diff
@@ -43,68 +43,122 @@ The main convention is use the role of each component to indicate the function
+ * mirror-xy: To perform the fine adjustments of the position of the parabolic mirror.
+   It has two axes: x and y.
+ * lens-mover: Allows to position lens 1 within the optical path perpendicular to the optical axis.
+   Lens 1 focuses the incoming collimated light. It has an axis: x.
+ * lens-switch: Switches lens 2 between two positions (on: within light path; off: outside of light path).
+   Lens 2 is used to further focus the light coming from lens 1. It has an axis: x or rx.
+ * brightlight: Is used to calibrate the position offset between the two detectors, the grating offset and
+   the focus (mirror) within the spectrograph.
+ * polarization analyzer: It is used to switch the quarter wave plate and the linear polarizer to well
+   specified relative positions to analyze the polarization grade of the emitted light. It has one axis: pol.
+ * quarter wave plate: First polarizer of the polarization analyzer. It has one axis: rz.
+ * linear polarizer: Second polarizer of the polarization analyzer. It has one axis: rz.
+ * slit-in-big: Slit is used to tune the spectral resolution. It can be switched between position "on",
+   which is completely opened, and position "off", which is nearly closed. If switched to "off" axis "slit-in"
+   in spectrograph is initiated, which allows a fine tuning of the slit size.
+ * filter: Emission filter on the optical path of the SPARCv2 to select a specific wavelength band.
+   It has an axis: band.
+ * spectrograph: Controls the actuators related to spectrometry. It controls the spectrograph
+   components slit and grating turret.
+   It has an axis: wavelength.
+   It has the optional axes: grating, slit-in (independent of each other).
+   The grating turret can be either consisting of two mirrors (one on each side) or a mirror and
+   a grating. Then axis "grating" controls the switching between these two positions.
+   In combination with the axis "wavelength" the center wavelength of the grating can be selected.
+   If a mirror is selected on the grating turret within the optical path, the spectrograph is not
+   operated as a spectrograph in the classical sense anymore and the mandatory axis wavelength is 0.
+   The axis "slit-in" controls the fine adjustments of the slit. If the slit is switched "on" via "slit-in-big"
+   (completely open the slit), axis "slit-in" is forced to be completely opened.
+   If the slit is switched to "off" via "slit-in-big", fine adjustments of the slit can be conducted via
+   the axis "slit-in".
+ * focus: Changes the lens distance to the sample. It has one axis: z. # TODO which lens?? fixed in spectrograph?
+ * spec-det-selector: Mirror to switch between multiple detectors connected to a spectrograph.
+   It has an axis: rx.
+ * fiber-aligner: To move optical fiber, typically with axes "x" and "y". # TODO: explanation
+ * ar-spec-selector: Selector between AR/Spectrometer for the SPARC.
+   It has an axis: rx. # TODO: explanation
+ * stage: TODO??
```

Comment:
stage: already described as the first actuator.
scan-stage: optional fast and accurate moving stage used to move the sample during an acquisition instead of moving the e-beam. It should have axes x and y.